api/globalSearch/index.ts

```diff
@@ -58,14 +59,17 @@ export function generateServices(jurisdictions: Jurisdiction[]): GlobalSearchSer
   const globalSearchServiceIds = getConfigValue(GLOBAL_SEARCH_SERVICES);
   const globalSearchServiceIdsArray = globalSearchServiceIds.split(',');
 
+  const serviceNamesMappings = getConfigValue(SERVICE_NAME_MAPPINGS) as GlobalSearchService [];
+
   // Generate global search services
   const globalSearchServices: GlobalSearchService[] = [];
   globalSearchServiceIdsArray.forEach((serviceId) => {
     const jurisdiction = jurisdictions ? jurisdictions.find((x) => x.id === serviceId) : null;
     if (jurisdiction) {
       globalSearchServices.push({ serviceId: jurisdiction.id, serviceName: jurisdiction.name });
     } else {
-      globalSearchServices.push({ serviceId, serviceName: serviceId });
+      const serviceName = serviceNamesMappings.find((map) => map.serviceId === serviceId).serviceName;
```

Comment:
@udaydenduluri33 - what happens if `find()` returns `null`? Would it not cause an error?

Reply:
Thanks, @udaydenduluri33

Reply:
Fixed it now. Thanks